modules/cnv-subscribing-to-hco-catalog.adoc

```diff
@@ -34,9 +34,9 @@ list and choose the `openshift-cnv` namespace.
 `openshift-operators` namespace to watch and be made available to all namespaces
 in the cluster. This option is not supported for use with {CNVProductName}.
 You must only install the Operator in the `openshift-cnv` namespace.
-* *A specific namespace on the cluster* allows you to choose a specific, single
-namespace in which to install the Operator. The Operator will only watch and be
-made available for use in this single namespace.
+* *A specific namespace on the cluster* is what you must select.
+The Operator will only watch and be
+made available for use in the `openshift-cnv` namespace.
```

Comment:
Suggestion: remove this bullet point entirely instead of rephrasing it. 

Step 5a already clearly states that you must choose the *A specific namespace on the cluster* option and use the `openshift-cnv` namespace. The first bullet point in this warning is necessary to explain why the customer should _not_ choose the default option, but the second one is just informational. I don't think it's necessary, and if it's causing confusion, I'd rather take it out.

To be on the safe side, I would also suggest bolding the word *not* in the first bullet point. (`This option is *not* supported for use with {CNVProductName}.`)

Thoughts, @dankenigsberg?

Reply:
I like your idea of dropping the second point. Can you modify this pr yourself? I'm afk.

Reply:
@dankenigsberg I couldn't edit it directly because I'm not a maintainer, but I opened https://github.com/openshift/openshift-docs/pull/19004 and will have it merged. I am bringing this change into both CNV 2.1 and 2.2.

Reply:
@dankenigsberg The other PR has been merged; you can close this one. Thanks!